templates/common/fs_test.go

```diff
@@ -393,6 +461,20 @@ func TestCopyRecursive(t *testing.T) {
 			if diff := cmp.Diff(gotBackups, tc.wantBackups, CmpFileMode, cmpopts.EquateEmpty()); diff != "" {
 				t.Errorf("backups directory was not as expected (-got,+want): %s", diff)
 			}
+
+			if diff := cmp.Diff(hashes, tc.wantHashes); diff != "" {
+				t.Errorf("hashes were not as expected: (-got,+want): %s", diff)
+			}
 		})
 	}
 }
+
+func mustHexDecode(t *testing.T, hexStr string) []byte {
```

Comment:
Prefer tb

Reply:
Function is now gone with the switch to returning hex from copyRecursive